pkg/okctl/okctl.go

```diff
@@ -51,6 +51,9 @@ import (
 	"github.com/oslokommune/okctl/pkg/storage"
 )
 
+// ErrorEnvironmentNotFound indicates that the requested environment isn't available
+var ErrorEnvironmentNotFound = errors.New("environment not found")
```

Comment:
Consider adding more info: "environment not found: dev".

Even better:

"environment not found: dev. Possible values are: prod, staging"

which I guess you can find from o.RepoState.Clusters like you did for apply cluster.

Reply:
Possible values are on line 50 in applyapplication.go

I'll add the specified invalid environment as well

Reply:
Right. This is done. Please re-review @yngvark.

Idiomatic Go error handling is not my strong suit, so please do come with tips if you have any